modules/rosa-setting-up-environment.adoc

```diff
@@ -38,7 +38,14 @@ This user has `Programmatic` access enabled and the `AdministratorAccess` policy
 +
 Specify the correct `aws_access_key_id` and `aws_secret_access_key` in the `.aws/credentials` file. See link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html[AWS Configuration basics] in the AWS documentation.
 
-.. Optional: Set a default AWS region. `rosa` evaluates regions in the following priority order:
+.. Optional: Set a default AWS region.
++
+[NOTE]
+====
+It is recommended to set the default AWS region using the environment variable.
+====
++
+`rosa` evaluates regions in the following priority order:
```

Comment:
ISG recommends against using lowercase at the start of a sentence. Suggest something like:

> The `rosa` CLI evaluates regions in the following priority order:

Also, I'm confused with this part because it falls under a sub-task of "Install AWS CLI". But the formatting suggests something about the `rosa` CLI or command. So maybe instead you want to say "The ROSA service evaluates regions..." here as is done in the previous step ("Enable the ROSA service in the AWS Marketplace")?

Reply:
Good idea. Done. Thanks.